api/tests/Feature/CountPoolCandidatesByPoolTest.php

```diff
@@ -828,4 +828,60 @@ public function testAdditionalAvailabilityScopes()
             ],
         ]);
     }
+
+    // asserts placed term/indeterminate candidates can appear in this query, can't check by id though so check that 2 out of 3 appear
+    public function testPlacedSuspendedNotSuspendedCandidates()
+    {
+        $itPool = Pool::factory()->create([
+            ...$this->poolData(),
+            'publishing_group' => PublishingGroup::IT_JOBS->name,
+        ]);
+        PoolCandidate::truncate();
+        $candidate = PoolCandidate::factory()->availableInSearch()->create([
+            'pool_id' => $itPool,
+            'pool_candidate_status' => PoolCandidateStatus::PLACED_TERM->name,
+        ]);
+        $candidate->suspended_at = null;
```

Comment:
Why do you change `suspended_at` and resave after the factory call?

Reply:
Same comment about the other pool candidate tests.

Reply:
I thought the configure states did something to interfere, guess not
https://github.com/GCTC-NTGC/gc-digital-talent/pull/12330/commits/9af9134ea554e83f7db886038b86be858c05f194